docs/users/installation.md

```diff
@@ -9,7 +9,7 @@ title: Installation
 
 **Java LTS (8, 11, 17 or 21)**
 
-**Scala 2.12, 2.13 or 3.x** (all rules are not available for Scala 3.x)
```

Comment:
only `ExplicitResultTypes` remains unsupported, and it's documented in rule docs, so I think we can drop this verbose disclaimer